src/openforms/submissions/tests/test_admin.py

```diff
@@ -143,3 +148,71 @@ def test_change_view_displays_logs_if_not_avg(self):
 
         # avg log not visible
         self.assertNotContains(response, avg_log.get_message())
+
+
+@disable_2fa
+class TestLogicEvaluated(VariablesTestMixin, WebTest):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        cls.submission = SubmissionFactory.from_data(
+            {
+                "firstname": "foo",
+                "birthdate": "2022-06-20",
+            },
+        )
+
+        cls.admin_user = SuperUserFactory.create()
+        cls.anonymous_user = UserFactory.create()
+
+    def test_invalid_permission_view(self):
+        self.app.get(
+            reverse("admin:logs-evaluated-logic", args=(self.submission.pk,)),
+            user=self.anonymous_user,
+            status=status.HTTP_403_FORBIDDEN,
+        )
+
+    def test_valid_permission_view(self):
+        self.app.get(
+            reverse("admin:logs-evaluated-logic", args=(self.submission.pk,)),
+            user=self.admin_user,
+            status=status.HTTP_200_OK,
+        )
```

Comment:
this needs a bit more nuance - you also want to test the following user types:

* authenticated user, not staff, no permissions
* authenticated user, staff, no permissions
* authenticated user, no staff, with permissions
* authenticated user, staff, with permissions (but not superuser)